man/nudoku.6

```diff
@@ -57,6 +57,10 @@ Display version information.
 .BR \-c
 Do not use colors. Black and white mode. Some terminals don't support colors, use this option in this case.
 
+.BR \-o
+Output stream.
+Puts a single sudoku into the standard output. This format is the same as the one read by -s.
```

Comment:
"Print a single sudoku onto standard output. Notation is in stream format like used by -s."